### ISSUE TITLE
misc

### ISSUE THREAD

**Issue by agramfort** [ramp-kits/dreem#1]
@lucyleeow you should see how to activate CIs on this repo.

also I saw a few issues on the starting kit eg
sk-learn -> scikit-learn
use of accuracy and not roc-auc in the cross-val 

nice work !

**Comment by lucyleeow** [comment]
- Thanks, I will fix that.
- I used accuracy simply because the original challenge on 'Challenge data' used this, and did not look into. Looking all at the available [evaluation metrics](https://scikit-learn.org/stable/modules/model_evaluation.html), why is ROC-AUC preferred in cross-val? (FYI - there are 3 prediction labels (0 - no slow oscillation, 1 - slow oscillation low amp, 2 - slow oscillation high amp)).  
- Will look into CI

**Comment by agramfort** [comment]
true I don't know where I dreamt the "roc-auc"...

also remove __pycache__ in
https://github.com/ramp-kits/dreem/tree/master/submissions/starting_kit

**Comment by lucyleeow** [comment]
- Removed pycache
- I just found out that [roc-auc](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score) works for multiclass classification - it does pairwise comparison (one class vs all others) and then averages them in some way. Not sure what would be the best metric though...?

**Comment by agramfort** [comment]
don't change the metric used in the other challenge. So we'll be able to
compare

>
